src/emulator/storage/apis/firebase.ts

```diff
@@ -128,6 +130,48 @@ export function createFirebaseEndpoints(emulator: StorageEmulator): Router {
     return res.json(new OutgoingFirebaseMetadata(metadata));
   });
 
+  firebaseStorageAPI.post(`/b/:bucketId/o/:objectId[(:)]generateSignedUrl`, async (req, res) => {
```

Comment:
Right now I think this will create signed urls for objects that don't exist. We should return a 404 in that case.